packages/jsii/lib/assembler.ts

```diff
@@ -467,14 +479,18 @@ export class Assembler implements Emitter {
       throw new Error('Oh no');
     }
 
-    for (const decl of type.symbol.declarations) {
+    const allDeclarations: Array<{ decl: ts.Declaration, type: ts.InterfaceType | ts.BaseType }>
+      = type.symbol.declarations.map(decl => ({ decl, type }));
+    // Considering erased bases' declarations, too, so they are "blended in"
+    erasedBases.forEach(base => allDeclarations.push(...base.symbol.declarations.map(decl => ({ decl, type: base }))));
```

Comment:
Can you convert to a for-loop? Makes it much easier for me to read :-)... In general, would appreciate if the code would be a little less condensed (I am old...)

Reply:
I value vertical context, which is why I don't put a lot of vertical whitespace instinctively...

Reply:
I value both. That’s why I like empty lines